index.js

```diff
@@ -0,0 +1,75 @@
+// EXERCISE 1: Selecting Elements
+// Select the element with the ID 'mainTitle' and log it to the console.
+
+let mainTitle = document.getElementById("mainTitle");
+console.log(mainTitle)
+
+
+// Exercise 2: Changing HTML Content
+// TODO: Select all paragraph elements and change the content of the second paragraph to "This paragraph has been changed."
+
+let paragraph = document.getElementsByClassName("paragraph");
+paragraph[1].innerHTML = "<p>This paragraph has been changed.</p>";
+
+// EXERCISE 3: Changing CSS Styles
+// Change the background color of the element with the ID 'content' to 'lightgreen'.
+
+let content = document.getElementById("content");
+content.style.backgroundColor = "lightgreen";
+
+// Exercise 4: Adding/Removing Classes
+// TODO: Add the "highlight" class to the first paragraph
+// TODO: Remove the "highlight" class from the first paragraph after 3 seconds
+
+paragraph[0].classList.add("highlight");
+setTimeout(function () { // This function adds 3 seconds delay to removing the class "highlight"
+    paragraph[0].classList.remove("highlight");
+}, 3000);
+
+ // EXERCISE 5: Creating and Appending Elements
+ // Create a new <button> element with the text 'Click Me'.
+ // Append this button to the element with the ID 'content'.
+
+ let button = document.createElement("button");
+ button.textContent = "Click Me";
+ document.getElementById("content").appendChild(button); // Adding a button at the end of the element with id content.
+
+ // EXERCISE 6: Removing Elements
+ // Select the element with the ID 'result' and remove it from the DOM.
+
+ let result = document.getElementById("result");
+ result.remove();
+
+ // EXERCISE 7: Handling Events
+ // Add a 'click' event listener to the button created in Exercise 5.
+ // When clicked, the button should alert "Button Clicked!".
+
+ button.addEventListener("click", function () {
+
```

Comment:
- [ ] Remove extra line

Reply:
Do I have to remove all the blank lines inside the functions, or only the 48?

Reply:
Only the ones marked in the tasks, not all of them.